### PR TITLE
[Extra/rest 2.5] add relayer stat

### PR DESCRIPTION
**Pull request by satawatnack**
#### ADD IBC relayer stat to support this implementation 

Table relayer_tx_stat_days

| Column 1 | Type |  |
| -------- | -------- | -------- |
| date     | CustomDate     | PK     |
| port     | String     | PK, FK   |
| channel     | String     | PK, FK     |
| address     | String     | PK     |
| ibc_received_txs     | Integer     |     |
| last_update_at     | CustomDateTime     |     |

> Note: 
> ```
> CREATE TABLE RELAYER_TX_STAT_DAYS(
>   DATE DATE NOT NULL,
>   PORT VARCHAR (20) NOT NULL,
>   CHANNEL VARCHAR (20) NOT NULL,
>   ADDRESS  VARCHAR (255) NOT NULL,
>   IBC_RECEIVED_TXS INT,
>   LAST_UPDATE_AT TIMESTAMP,
>   CONSTRAINT PK_RELAYER_STATS PRIMARY KEY (DATE,PORT,CHANNEL,ADDRESS),
>   FOREIGN KEY (PORT,CHANNEL) REFERENCES CHANNELS(PORT,CHANNEL)
>);
> ```

## Please ensure the following requirements are met before submitting a pull request:

- [x] The pull request is targeted against the correct target branch
- [ ] The pull request is linked to an issue with appropriate discussion and an accepted design OR is linked to a spec that describes the work.
- [x] The pull request includes a description of the implementation/work done in detail.
- [ ] The pull request includes any and all appropriate unit/integration tests
- [ ] You have added a relevant changelog entry to `CHANGELOG_UNRELEASED.md`
- [x] You have re-reviewed the files affected by the pull request (e.g. using the `Files changed` tab in the Github PR explorer)

### CODE REVIEW

File: flusher/flusher/db.py [hunk 1]
```diff
@@ -462,3 +462,15 @@ def Column(*args, **kwargs):
...
+
+relayer_tx_stat_days = sa.Table(
+    "relayer_tx_stat_days",
+    metadata,
+    Column("date", CustomDate, primary_key=True),
+    Column("port", sa.String, primary_key=True),
+    Column("channel", sa.String, primary_key=True),
+    Column("address", sa.String, primary_key=True),
+    Column("ibc_received_txs", sa.Integer),
+    Column("last_update_at", CustomDateTime),
```
Comment (colmazia) [inline flusher/flusher/db.py:474]: So, it is not a primary key

File: hasura/hasura-metadata/tables.yaml [hunk 2]
```diff
@@ -464,6 +474,20 @@
     - name: oracle_script
       using:
         foreign_key_constraint_on: oracle_script_id
+- table:
+    name: relayer_tx_stat_days
+    schema: public
+  object_relationships:
+    - name: channelByChannelPort
```
Comment (RogerKSI) [inline hasura/hasura-metadata/tables.yaml:481]: should be snakecase?

File: flusher/flusher/db.py [hunk 3]
```diff
@@ -462,3 +462,15 @@ def Column(*args, **kwargs):
     Column("order", sa.String),
     Column("last_update", CustomDateTime, index=True),
 )
+
+relayer_tx_stat_days = sa.Table(
```
Comment (RogerKSI) [inline flusher/flusher/db.py:466]: please help add sql command to create this table as well.

**Comment by RogerKSI** [comment]
Note: merge after chain upgrade v2.5.

**Review by colmazia — APPROVED** [review]

**Comment by RogerKSI** [comment]
LGTM. let me test on testnet.

**Review by RogerKSI — APPROVED** [review]

**Pull request merged**